src/components/buttonFavorites/ButtonFavorites.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useContext, useState } from "react";
+import VideoContext from "../../providers/VideoContext";
+import * as Style from "../styles/styleSheet.js";
+
+const ButtonFavorites = () => {
+  const { userSession } = useContext(VideoContext);
+  const { favoriteVideosMapUser } = useContext(VideoContext);
+  const { videoPropperties } = useContext(VideoContext);
+  const [textButton, setTextButton] = useState("Add to favorites");
```

Comment:
I think you could join the declaration of all constants coming from VideoContext, like:

`const { userSession, favoriteVideosMapUser , videoPropperties } = useContext(VideoContext);`